homeassistant/components/jellyfin/entity.py

```diff
@@ -32,3 +34,44 @@ def __init__(
             name=self.coordinator.server_name,
             sw_version=self.coordinator.server_version,
         )
+
+
+class JellyfinClientEntity(JellyfinEntity):
+    """Defines a base client Jellyfin entity."""
+
+    session_data: dict[str, Any] | None
```

Comment:
So when can this be `None`? We always get session data from the constructor right?

Reply:
Good question, in `JellyfinMediaPlayer` it can be set to `None` inside `_handle_coordinator_update`.

Which begs the question: should `JellyfinRemote` also have a `_handle_coordinator_update` that updates `self.session_data`? :thinking:

Reply:
Handled in 9eb04ad.